server/table_namespace_classifier.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"github.com/pingcap/pd/server/core"
+	"github.com/pingcap/pd/server/namespace"
+)
+
+// Namespace defines two things:
+// 1. relation between a Name and several tables
+// 2. relation between a Name and several stores
+// It is used to bind tables with stores
+type Namespace struct {
```

Comment:
Seem it is TableNamespace, not a Namespace.

Reply:
Will there be other namespace? How about change `table_namespace_classifier.go` to `namespace_classifier.go`?

Reply:
We may have other namespace implementation later. So we can't assume there is only one table namespace.